sqlglot/expressions.py

```diff
@@ -913,6 +913,9 @@ def where(
         Returns:
             Select: the modified expression.
         """
+        expressions = [exp for exp in expressions if exp is not None and exp != ""]
```

Comment:
I think this should work the same:

```python
expressions = [exp for exp in expressions if exp]
```

It's a little less explicit, but I don't think there are other arguments we're expecting that could evaluate to False?

Reply:
Also, do we want the same thing for `having`? Maybe we should move this into `_apply_conjuction_builder`?

Reply:
yeah you are right but see my test, I am guessing `where(False)` is valid

Reply:
in his test he allows where(False) which is different, although i wonder if this is correct

Reply:
why would you ever do where False, i think this is tricky, and probably not necessary

Reply:
it is correct, tried it in postgres

Reply:
you are right I can move into the conjunction builder, having should do the same